shared/config/chapters.ts

```diff
@@ -9,3 +11,10 @@ export enum FsdChapters {
   BackendJavascript = 'backend-javascript',
   Python = 'python'
 }
+
+export interface Chapter {
+  dashedName: string;
+  comingSoon?: boolean;
+  modules: Module[];
+  chapterType?: string;
+}
```

Comment:
Bit of nit, but it feels like this (and Module) should be reused in the client if we're going to put them in shared

Reply:
Sure, that makes sense.